docs_user/modules/proc_adopting-networker-services-to-the-data-plane.adoc

```diff
@@ -270,32 +256,32 @@ spec:
   - pre-adoption-validation
 EOF
 ----
-+
-Wait for the validation to finish.
+
+.. Wait for the validation to finish.
```

Comment:
Could combine these 2 sub steps into something like:
"When the validation is finished, confirm that the status of the Ansible EE pods is `Completed`:"

Could you re-use the similar section above for this?